core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.ts

```diff
@@ -232,6 +263,29 @@ export class TranslationSuggestionReviewModalComponent implements OnInit {
     }
   }
 
+  calculatePanelHeight(): void {
```

Comment:
Maybe this should be something like `computePanelOverflowState` as the panel heights are more implementation details.

Reply:
Done